### ISSUE TITLE
Aliases - more flexibility when setting them; specifically to allow "" surrounding values?

### ISSUE THREAD

**Issue by shevegen** [cmderdev/cmder#1727]
### Purpose of the issue
- [ ] Bug report (encountered problems/errors)
- [x ] Feature request (request for new functionality)
- [ ] Question

### Description of the issue 

Hey there. This is a suggestion to make the aliases a bit more flexible for cmder.

For example, I believe this alias works:

    alias 'fo'=echo Foo bar

But I believe this one here does not work:

    alias 'fo'="echo Foo bar"

(I can not say for 100% because I am on a linux machine right now; cmder is at my university workspace to which I only have physical access during non-holidays, and right now we have holidays here. But I remember that my old aliases all do not work as-is via cmder and I checked that the reason for this were the " quotes.)

So in other words - to change the alias-handling part of cmder to also allow for "" characters.

Would this be a possibility? Perhaps there is something that does not allow for this, but if it were syntactically possible, this would be great. Then I could use all my bash-aliases as-is via cmder too. (They are enclosed in either " or ', depending on whether they include a " or ' internally)

**Comment by daxgames** [comment]
cmder is not linux and windows does not use `alias` cmd from linux.  Cmder provides a method of simulating `alias` using `doskey` in its default `cmd::Cmder` sessions.  You will need to add your aliases to cmder using the doskey syntax or you might consider using your aliases as is in a cmder `bash::bash` session.
